src/index.html

```diff
@@ -9,6 +9,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+
+      <a href="#home">
```

Comment:
```suggestion
    <header class="header">
      <a href="#home">
```
Don't add new lines between parent and child elements